bugreportutils/bugreportutils.go

```diff
@@ -128,15 +144,22 @@ type MetaInfo struct {
 	SdkVersion       int
 	BuildFingerprint string
 	ModelName        string
-	Sensors          map[int32]SensorInfo
+	Sensors          map[int32]*SensorInfo
```

Comment:
I'd consider if avoiding this change makes sense.

Reply:
Currently I have a map[int32]*SensorInfo, this allows me to directly modify the struct that is referenced by the pointer. So suppose I am updating the name of sensor with handle 1, I can just do:
```
sensor := map[1]
sensor.Name = "newName"
```
However with the old map[int32]SensorInfo type, to accomplish the same thing, I need to:
```
sensor := map[1]
sensor.Name = "newName"
map[1] = sensor
```
Since accessing the sensorInfor in the map gives me a copy of the value, and I need to overwrite the copy again.

Reply:
But you can always use map[1].Name without any impact, I think... & where is the code you are using that is impacted in this change.

I'm worried about this as when we start merging this back, similar questions might be asked.

Reply:
In golang, this following line does not work:
```
map[1].Name := "newName"
```
The compiler will throw a panic warning to this. I will stick with the pointer struct for now.

Reply:
I totally agree that the same question might be asked again when we start merging this back. Also considering that this change may impact other files, I will change it back to the original struct rather than using the pointer in the map.